recipes/cate-cli/meta.yaml

```diff
@@ -0,0 +1,142 @@
+{% set name = "cate-cli" %}
+{% set version = "2.0.0.dev10" %}
+{% set sha256 = "9ad7f8bfbff8ae22788efbc94a66f038f79ad2ee47ba786fdc9e4c9f977f3bc2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CCI-Tools/cate/archive/v{{ version }}.tar.gz
```

Comment:
Is it possible to download the other files that are bundled in this recipe as well?  
Conda allows you to specify multiple urls that will get downloaded.

Reply:
Hi! Thank you for the review, but I am not clear which other files you referred to? Do you mean the dependencies?

Reply:
Things like recipes/cate-cli/cate-cli.app/Contents/Info.plist the icons the pngs etc